packages/client/src/App.re

```diff
@@ -1,50 +1,46 @@
 type state = {
   route: Routing.Route.t,
   authToken: option(string),
+  routingCount: int,
```

Comment:
Why is this needed? Can you add a comment to the code or similar? I imagine it could also have a more specific name, I really don't understand what this means.

Reply:
Thanks for your question, have changed this a bit, moved this var into "Redirect" component and renamed it into "countOfRedicrectings". This var need to prevent "user cannot go from profile page".

Reply:
Maybe change the name to "redirectionCount" or at least "countOfRedirections", to fix spelling?

Reply:
Thanks, fixed.